app/views/application/_subnavigation.html.erb

```diff
@@ -19,7 +19,7 @@
       <%= link_to "Setting Up", setup_instructions_path, class: active_tab(setup_instructions_path) %>
       <%= link_to 'Organisation', organisation_path(current_organisation), class: active_tab("organisation") %>
       <%= link_to "Team members", team_members_path, class: "#{active_tab("team")}" + "#{active_tab("user")}"%>
-      <%= link_to "Settings", ips_path, class: active_tab("ips") %>
+      <%= link_to "IP's", ips_path, class: active_tab("ips") %>
```

Comment:
I'd recommend against using apostrophes for pluralising abbreviations. It's technically acceptable, but [modern style is to omit](https://english.stackexchange.com/questions/55970/plurals-of-acronyms-letters-numbers-use-an-apostrophe-or-not), and I feel it's clearer without.

```suggestion
      <%= link_to "IPs", ips_path, class: active_tab("ips") %>
```